packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -37,12 +39,32 @@ export interface BucketDeploymentProps {
    * @default true - when resource is deleted/updated, files are retained
    */
   readonly retainOnDelete?: boolean;
+
+  /**
+   * The CloudFront distribution using the destination bucket as an origin.
+   * Files in the distribution's edge caches will be invalidated after
+   * files are uploaded to the destination bucket.
+   *
+   * @default No invalidation occurs
+   */
+  readonly distribution?: cloudfront.IDistribution;
+
+  /**
+   * The file paths to invalidate in the CloudFront distribution.
+   *
+   * @default All files under the destination bucket key prefix will be invalidated.
```

Comment:
Set the default "value" to "-" (it's a convention we use in case there is no specific value to provide)